test/unit/test_config.py

```diff
@@ -0,0 +1,122 @@
+import os
+from shutil import copyfile
+from tempfile import TemporaryDirectory
+
+import pytest
+import yaml
+
+from ozy import OzyError
+from ozy.config import safe_expand, parse_ozy_conf, resolve, apply_overrides, get_user_conf_file, load_config
+
+
+def test_parse_ozy_conf():
+    sample_config_file = "conf/sample-team-conf.yaml"
```

Comment:
Maybe consider defining this relative to this script's location? Currently this forces you to run the test from the project root (which might be inconvenient in IDEs for example?)

Reply:
I run my tests from pycharm and by habit always set the current working directory to be the project root, so ... no inconvenience for my usage of pycharm ;-) I was thinking that this file might actually be better if it were copied into this test directory and used just for tests, as right now its doing dual duty (sample config file _and_ test input file)...